### PR TITLE
Change the version information of the node to kwok{{ Version }}

### LINKED ISSUE #395: [kwok] Change version of node information
### What would you like to be added?

Change the version information of the node to `kwok{{ kwokVersion }}`

### Why is this needed?

The version information of the node is `fake` by default,

### PR DESCRIPTION
**Pull request by HANXP-1**
#### Which issue(s) this PR fixes:

Fixes #395

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by wzshiming** [comment]
Please sign the CLA

File: stages/node-fast.yaml [hunk 1]
```diff
@@ -86,6 +86,7 @@ spec:
         bootID: {{ with .bootID }} {{ . }} {{ else }} "" {{ end }}
         containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "" {{ end }}
         kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "" {{ end }}
+        kwokVersion: {{ with .kwokVersion }} {{ . }} {{ else }} "fake" {{ end }}
         kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "fake" {{ end }}
         kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "fake" {{ end }}
```
Comment (wzshiming) [inline stages/node-fast.yaml:91]:
```suggestion
        containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
        kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
        kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
        kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
```

File: site/content/en/docs/user/kwok-manage-nodes-and-pods.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ status:
     bootID: ""
     containerRuntimeVersion: ""
     kernelVersion: ""
+    kwokVersion: fake
```
Comment (wzshiming) [inline site/content/en/docs/user/kwok-manage-nodes-and-pods.md:64]:
```suggestion
```

**Review by wzshiming — CHANGES_REQUESTED** [review]

File: stages/node-fast.yaml (hunk 1 above)
Comment (Hanvans) [reply stages/node-fast.yaml:91]: ok，thanks.

**Comment by wzshiming** [comment]
Please sign the CLA

**Comment by hanxp2** [comment]
> [CLA Missing ID](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/20677499/518708142/407/#/?version=2) [CLA Not Signed](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/20677499/518708142/407/#/?version=2)
> 
> * ❌ The commit ([3406ac5](https://github.com/kubernetes-sigs/kwok/commit/3406ac5be2f19114ff1fee4096a042d975c2243f)). This user is missing the User's ID, preventing the EasyCLA check. [Consult GitHub Help](https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user) to resolve.For further assistance with EasyCLA, [please submit a support request ticket](https://jira.linuxfoundation.org/servicedesk/customer/portal/4).

**Comment by wzshiming** [comment]
> Missing CLA Authorization.

It doesn't look like it was signed, how did you do it?

File: stages/node-fast.yaml [hunk 3]
```diff
@@ -84,10 +84,10 @@ spec:
...
         architecture: {{ with .architecture }} {{ . }} {{ else }} "amd64" {{ end }}
         bootID: {{ with .bootID }} {{ . }} {{ else }} "" {{ end }}
-        containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "" {{ end }}
-        kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "" {{ end }}
-        kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "fake" {{ end }}
-        kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "fake" {{ end }}
+        containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
+        kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
+        kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
+        kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
```
Comment (wzshiming) [inline stages/node-fast.yaml:90]: This also requires the implementation of `Version` template function

**Review by wzshiming — CHANGES_REQUESTED** [review]

File: stages/node-fast.yaml (hunk 3 above)
Comment (hanxp2) [reply stages/node-fast.yaml:90]: Where I can define ther template function of `Version`？Can you give me an example of something like this.

**Comment by wzshiming** [comment]
https://github.com/kubernetes-sigs/kwok/blob/4a43d577d786f8a17e3345bd1e6f6d3907cfe18f/pkg/kwok/controllers/controller.go#L60

``` golang
"Version": func() string {
    return consts.Version
},
```

**Comment by hanxp2** [comment]
I received an email from the Cloud Native Computing Foundation (CNCF). it mentioned that The CLA has now been signed.

**Comment by wzshiming** [comment]
Please make sure that your email address is the same as the one in the commit

File: stages/node-fast.yaml (hunk 3 above)
Comment (wzshiming) [inline stages/node-fast.yaml:90]:
```suggestion
        containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "kwok-{{ Version }}" {{ end }}
        kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "kwok-{{ Version }}" {{ end }}
        kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "kwok-{{ Version }}" {{ end }}
        kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "kwok-{{ Version }}" {{ end }}
```

**Comment by wzshiming** [comment]
> by hanxiaopeng, 03/23/2023 10:58 AM
> committed by hanxp, 03/24/2023 01:43 PM

This push and committee are not the same account?

**Comment by HANXP-1** [comment]
> > by hanxiaopeng, 03/23/2023 10:58 AM
> > committed by hanxp, 03/24/2023 01:43 PM
> 
> This push and committee are not the same account?

I closed this pr and created a new commit https://github.com/kubernetes-sigs/kwok/pull/416. Thank you very much for my help!

**Pull request closed without merging**